ksqldb-execution/src/main/java/io/confluent/ksql/execution/windows/WindowTimeClause.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.windows;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.errorprone.annotations.Immutable;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Pojo for a time clause added to a window expression
+ */
+@Immutable
+public class WindowTimeClause {
```

Comment:
why not `java.time.Duration`? seems to do the same thing but with more utility methods. Happy either way

Reply:
we need this class mostly because we print the expressions in the `toString` method.. If we eagerly convert to Duration, we lose information on the actual SQL clause i.e `60 SECONDS` vs `1 MINUTE`